lib/minesweeper_2pl/board.rb

```diff
@@ -1,4 +1,31 @@
 module Minesweeper_2pl
+
   class Board
+    attr_accessor :size, :bomb_count, :bomb_positions, :positions, :row_size
```

Comment:
Do you need to set these attributes once you've created a `Board`, or is the `Board` an immutable data object that shuffles some bombs into random locations and won't change? If you don't need to set values, consider using `attr_reader`s instead.